app/src/main/java/com/asfoundation/wallet/analytics/RakamAnalytics.kt

```diff
@@ -0,0 +1,111 @@
+package com.asfoundation.wallet.analytics
+
+import android.app.Application
+import android.content.Context
+import android.util.Log
+import com.asf.wallet.BuildConfig
+import com.asfoundation.wallet.identification.IdsRepository
+import com.asfoundation.wallet.logging.Logger
+import com.asfoundation.wallet.logging.RakamReceiver
+import io.rakam.api.Rakam
+import io.rakam.api.RakamClient
+import io.rakam.api.TrackingOptions
+import io.reactivex.Single
+import io.reactivex.schedulers.Schedulers
+import org.json.JSONException
+import org.json.JSONObject
+import java.net.MalformedURLException
+import java.net.URL
+
+class RakamAnalytics(private val context: Context, private val idsRepository: IdsRepository,
+                     private val logger: Logger) :
+    AnalyticsSetUp {
+  private val rakamClient = Rakam.getInstance()
+
+  companion object {
+    private val TAG = RakamAnalytics::class.java.simpleName
+  }
```

Comment:
Move companion object to bottom of file

Reply:
Agreed to be used on the top of the file